CoinDock-api/app/Models/V1/User.php

```diff
@@ -272,6 +279,29 @@ public function signUp()
         return $this->hasOne(Signup::class);
     }
 
+    public function updateProfile(updateProfileRequest $request, User $user)
+    {
+        $updatedUser = $request->all();
+        $user->update($updatedUser);
+        return response([
+            'message' => 'Profile updated succesfully',
+            'results'=>[
+                'user'=>new UserResource($user)
+            ]
+        ],Response::HTTP_OK );
```

Comment:
Above return, the response should be handled by the controller.

Reply:
done